.github/workflows/ci.yml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches:
     - main
+    - 'release/**'
```

Comment:
I was going to ask whether the `**` is a valid expression, and yep, this is per https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters.